src/test/java/serenityswag/inventory/AddProductsActions.java

```diff
@@ -0,0 +1,18 @@
+package serenityswag.inventory;
+
+import net.serenitybdd.core.pages.WebElementFacade;
+import net.serenitybdd.core.steps.UIInteractionSteps;
+import net.thucydides.core.annotations.Step;
+
+import java.util.List;
+
+public class AddProductsActions extends UIInteractionSteps {
+
+    ProductList productList;
+    @Step("Add Products'")
+    public Integer forAllAvailableItems() {
```

Comment:
```suggestion
    public int forAllAvailableItems() {
```
you usually use the primitive version